tests/test_app/test_data_store_new.py

```diff
@@ -208,108 +208,30 @@ def test_append(w_dstore):
-    expect = Path( write_dir / "brca1.fasta").read_text()
-
-    identifier = make_identifier(w_dstore, "brca1.fasta", absolute=True)
-    abs_id = w_dstore.write(identifier, expect)
-    md5 = "05a7302479c55c0b5890b50f617c5642"
-    assert w_dstore.md5(abs_id) == md5
-    assert w_dstore[0].md5 == md5
-
-    # does not have md5 if not set
-    identifier = make_identifier(w_dstore, "brca1.fasta", absolute=True)
-    abs_id = w_dstore.write(identifier, expect)
-    got = w_dstore.md5(abs_id, force=False)
-    assert got is None
-    # but if you set force=True, you get it
-    md5 = "05a7302479c55c0b5890b50f617c5642"
-    got = w_dstore.md5(abs_id, force=True)
-    assert got == md5
-
-
-def test_add_file():
-    """correctly add an arbitrarily named file"""
-    data = Path(f"data{os.sep}brca1.fasta").read_text()
-
-    log_path = os.path.join(dirname, "some.log")
-    with open(log_path, "w") as out:
-        out.write("some text")
-
-    path = os.path.join(dirname, basedir)
-    dstore = DataStoreDirectory(path, suffix=".fa", create=True)
-    _ = dstore.write("brca1.fa", data)
-    dstore.add_file(log_path)
-    assert "some.log" in dstore
-    assert os.path.exists(log_path)
-
-    log_path = os.path.join(dirname, "some.log")
-    with open(log_path, "w") as out:
-        out.write("some text")
-
-    path = os.path.join(dirname, basedir)
-    dstore = DataStoreDirectory(path, suffix=".fa", create=True)
-    _ = dstore.write("brca1.fa", data)
-    dstore.add_file(log_path, cleanup=True)
-    assert "some.log" in dstore
-    assert not os.path.exists(log_path)
-
-def test_make_identifier():
-    """correctly construct an identifier for a new member"""
-
-    if dirname.startswith(f".{os.sep}"):
-        dirname = dirname[2:]
-
-    path = os.path.join(dirname, basedir)
-    base_path = path.replace(".zip", "")
-    dstore = DataStoreDirectory(path, suffix=".json", create=True)
-    name = "brca1.fasta"
-    got = make_identifier(dstore, name, absolute = True)
-    expect = os.path.join(base_path, name.replace("fasta", "json"))
-    assert got == expect
-
-    # now using a DataMember
-    member = DataMember(
-        os.path.join(f"blah{os.sep}blah", f"2-{name}"), None
-    )
-    got = make_identifier(dstore, member, absolute=True)
-    expect = os.path.join(base_path, member.name.replace("fasta", "json"))
-    assert got == expect
-
-
-def test_summary_logs():
-    ...
-
-def test_summary_not_completed():
-   ...
+    assert len(nc_dstore.not_completed) == 3
 
-'''
 
-# todo: check combinations of args to constructor
-# todo: new tests for describe and summary
+def test_no_not_completed_subdir(nc_dir, nc_dstore):
+    nc_dstore.drop_not_completed()
+    Path(nc_dstore.source / _NOT_COMPLETED_TABLE).rmdir()
+    assert not Path(nc_dstore.source / _NOT_COMPLETED_TABLE).exists()
+    expect = f"6x member DataStoreDirectory(source='{nc_dir}', members=[brca1.fasta, long_testseqs.fasta, formattest.fasta...)"
```

Comment:
this test is fragile in the sense that if we change (for example) the class name the test will break. I suggest changing to
```python
expect = "6x member"
assert repr(nc_dstore).startswith(expect)
```